src/ec/packages/ec-component-language-list/language-list-overlay.html.twig

```diff
@@ -80,31 +80,12 @@
         {{- _title -}}
       </div>
     </div>
-    <div class="ecl-row">
-      <div class="ecl-language-list__column ecl-col-12 ecl-col-lg-4 ecl-offset-lg-2">
-        <ul class="ecl-language-list__list">
-          {% for _item in _column1 %}
-            {% include './language-list-item.html.twig' with {
-              item: _item,
-              overlay: _overlay,
-              icon_path: _icon_path
-            } only %}
-          {% endfor %}
-        </ul>
-      </div>
-      <div class="ecl-language-list__column ecl-col-12 ecl-col-lg-4">
-        <ul class="ecl-language-list__list">
-          {% for _item in _column2 %}
-            {% include './language-list-item.html.twig' with {
-              item: _item,
-              overlay: _overlay,
-              icon_path: _icon_path
-            } only %}
-          {% endfor %}
-        </ul>
-      </div>
-    </div>
-  </div>
+
+    {% include './language-list.html.twig' with {
+      items: _items,
+      overlay: _overlay,
```

Comment:
The overlay attribute here is fixed and is positive: https://github.com/ec-europa/europa-component-library/blob/v2-dev/src/systems/ec/implementations/react/packages/language-list/src/LanguageListOverlay.jsx#L44

Reply:
Please also update corresponding documentation `/docs/overlay.md`.